package.json

```diff
@@ -29,7 +29,7 @@
     "prepare": "husky install",
     "typecheck": "tsc -b && tsc -b e2e",
     "validate": "run-p test lint typecheck test:e2e",
-    "validate:precommit": "run-p lint format:check typecheck",
+    "validate:precommit": "run-p lint format typecheck",
```

Comment:
Thanks for changing this! Thoughts on making the `lint` command automatically run `--fix`? Or maybe better to add a `lint:fix`. Either way, my thought is let's make `git commit` succeed as often as we can control 🙂

Reply:
If we do this, I suggest we change these commands to run serially instead of in parallel. Otherwise format and lint would both be modifying files at the same time. That could cause problems.

Reply:
Done. Let me know if you're good with it.